backend/src/services/RecaptchaService.ts

```diff
@@ -6,6 +6,9 @@ if (process.env.NODE_ENV !== "production") {
 }
 
 export async function verifyRecaptcha(token: any) {
+  if (token == "123") {
```

Comment:
what does this mean?

Reply:
bypass the recaptcha verification if user signs up through google lmao